selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceManagement.java

```diff
@@ -220,6 +280,235 @@ public void saveChangesSpace() {
+            info("Right arrow of the event calendar");
+            $(byXpath("//i[@class='uiIconMiniArrowRight uiIconLightGray']")).waitUntil(Condition.visible, Configuration.timeout);
+        }
+    }
+
+    /**
+     * Check Manager Title Dimensions
+     */
+    public void checkManagerTitleDimensions() {
+        assertEquals("The line height is correct", "20px", ELEMENT_SPACE_MANAGER_NAME.getCssValue("line-height"));
+        assertEquals("The font size is correct", "14px", ELEMENT_SPACE_MANAGER_NAME.getCssValue("font-size"));
+        assertEquals("The color is correct", "#333333", getCSSColor(ELEMENT_SPACE_MANAGER_NAME));
+        assertEquals("The font weight is correct", "400", ELEMENT_SPACE_MANAGER_NAME.getCssValue("font-weight"));
+        assertEquals("The font family is correct", "Helvetica, arial, sans-serif", ELEMENT_SPACE_MANAGER_NAME.getCssValue("font-family"));
+    }
+
+    /**
+     * Check Who's Online Portlet Dimensions
+     */
+    public void checkWhoIsOnlinePortletDimensions(String onlineUser) {
+        final SelenideElement onlineUserInSpace = $(byXpath("//h6[text()=\"Who's Online?\"]/following::img[@src=\"/portal/rest/v1/social/users/${onlineUserSpace}/avatar\"]".replace("${onlineUserSpace}", onlineUser)));
+        assertEquals("The line height is correct", "20px", onlineUserInSpace.getCssValue("line-height"));
+        assertEquals("The font size is correct", "14px", onlineUserInSpace.getCssValue("font-size"));
+        assertEquals("The font weight is correct", "400", onlineUserInSpace.getCssValue("font-weight"));
+        assertEquals("The font family is correct", "Helvetica, arial, sans-serif", onlineUserInSpace.getCssValue("font-family"));
+    }
+
+    /**
+     * Check Calendar Portlet
+     */
+    public void checkCalendarPortletInSpaceHomePage() {
+        final SelenideElement calendarCheck = $(byXpath("//div[@class='calendarPortlet']"));
+        assertEquals("The line height is correct", "20px", calendarCheck.getCssValue("line-height"));
+        assertEquals("The font size is correct", "14px", calendarCheck.getCssValue("font-size"));
+        assertEquals("The color is correct", "#333333", getCSSColor(calendarCheck));
+        assertEquals("The font weight is correct", "400", calendarCheck.getCssValue("font-weight"));
+        assertEquals("The font family is correct", "Helvetica, arial, sans-serif", calendarCheck.getCssValue("font-family"));
+    }
+
+    /**
+     * Check No Settings Button Display For Calendar
+     */
+    public void checkNoSettingsButtonDisplayForCalendar() {
+        final SelenideElement settingsButton = $(byXpath("//a[@data-original-title=\"Settings\"]"));
+        settingsButton.shouldNot(Condition.visible);
+    }
+
+    /**
+     * Change Status
+     */
+    public void changeStatus(String status) {
+        ELEMENT_CHAT_ICON_STATUS.waitUntil(Condition.visible, Configuration.timeout).click();
+        sleep(3000);
```

Comment:
The good practice is to use wait methods (like waitUntil). Can't we use it here ?

Reply:
Yes but we use here sleep after the button click

Reply:
We need to wait for what ? We should use until a given element appears for example.

Reply:
It's clear, done.